account_invoice_report_due_list/models/account_move.py

```diff
@@ -9,7 +9,9 @@ class AccountMove(models.Model):
     _inherit = "account.move"
 
     multi_due = fields.Boolean(string="Multiple date due", compute="_compute_multi_due")
```

Comment:
Sorry, this is the one needing the compute_sudo.

Reply:
Sorry, but I think that finally it is not necessary to make any changes in this addon.

Reply:
Yeah, sure. I was thinking in the same side effect on restricting accounts, but I'm thinking the problem will be in other place then.